packages/editor/src/components/projectEditor/ProjectEditor.tsx

```diff
@@ -243,7 +251,7 @@ export class ProjectEditor extends React.Component<IProps, IState> {
                                                 name='Console'
                                                 onClick={() => togglePanel(Panels.OutputLog)}
                                                 pillStatus={unreadRows && !this.isPanelOpen(Panels.OutputLog) ? '1' : '0'}
-                                                active={this.isPanelOpen(Panels.OutputLog)}
+                                                active={this.isPanelOpen(Panels.OutputLog) || this.hasRows()}
```

Comment:
This is causing an error, it's because the `hasRows()` function is being called on each render. We could move it into `componentDidUpdate` function and check when the rows has been updated and do the check in there.
 
![err](https://user-images.githubusercontent.com/17637244/77632943-ae147380-6f89-11ea-8536-9cafb0566fb5.png)